build/version.props

```diff
@@ -4,14 +4,4 @@
         <VersionPrefix>0.0.1</VersionPrefix>
         <VersionSuffix>preview</VersionSuffix>
     </PropertyGroup>
-
```

Comment:
I don't understand. Why would we want to remove that?

`<Version>` is what matters for .NET pack